.circleci/config.yml

```diff
@@ -48,13 +48,6 @@ jobs:
     <<: *defaults
     steps:
       - checkout
-      - run:
-          name: Unit Tests
```

Comment:
Should this be removed too?

Reply:
tests are run in drone now so this is redundant which is why I removed it.